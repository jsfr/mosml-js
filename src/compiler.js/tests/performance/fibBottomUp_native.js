var fib = function(n) {
  var fibcalc = function(n,x,y) {
    switch(n) {
      case 0.0: return x;
      default: return fibcalc(n-1.0,y,y+x);
    }
  }
  return fibcalc(n,0.0,1.0);
}

var a = fib(10.0);
var b = fib(50.0);
var c = fib(80.0);