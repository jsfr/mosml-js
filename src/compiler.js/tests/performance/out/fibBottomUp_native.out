==============================JS BEGIN==============================
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 1ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 1ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 1ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 1ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 1ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
naiveBottomUp_native: 0ms
==============================JS END================================
